argoverse/sensor_dataset_config.py

```diff
@@ -49,8 +47,27 @@ class SensorDatasetConfig:
     camera_sensors: SensorSuiteConfig = MISSING
 
 
-DATASET_NAME = "argoverse-v1.1"
+@dataclass(frozen=True)
+class ArgoverseConfig(SensorDatasetConfig):
+    RING_CAMERA_HEIGHT: int = 1200
+    RING_CAMERA_WIDTH: int = 1920
+
+    STEREO_CAMERA_HEIGHT: int = 2056
+    STEREO_CAMERA_WIDTH: int = 2464
+
+    dataset_name: str = "argoverse"
+    ring_cam_fps: int = 30
+    stereo_cam_fps: int = 5
 
-with hydra.initialize_config_module(config_module="argoverse.config"):
-    cfg = hydra.compose(config_name=f"{DATASET_NAME}.yaml")
-    ArgoverseConfig: SensorDatasetConfig = instantiate(cfg.SensorDatasetConfig)
+    camera_sensors: SensorSuiteConfig = SensorSuiteConfig(
```

Comment:
Repeated code is frustrating here ... but can't use `__post_init__` since the dataclass is frozen.

Reply:
I am missing something.. What's being repeated here? And can we not unfreeze the dataclass?

Reply:
Well, I just mean that we could generate this more succinctly in a loop. Maybe not an issue. We could unfreeze, but it's supposed to be immutable since it's a config.

Reply:
I think this is fine and not a major thing. Let's leave this for now.